src/components/RightSidebar/RightSidebar.vue

```diff
@@ -249,6 +259,19 @@ export default {
 			EventBus.$emit('show-settings', true)
 		},
 
+		handleUpdateDescription(payload) {
+			setTimeout(() => {
+				console.debug('payload', payload, 'waiting one second')
```

Comment:
should something happen after saving ?

usually I like to disable the fields while saving is in progress, but the RichContentEditable has no "disabled" property and I was told to set "contenteditable" to false for disabling.